File: src/codes/Users_Manual/geometry_4.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~using namespace autd3;
//~auto autd =
//~Controller::open({AUTD3{}}, link::Nop{});
//~{
auto dev = autd[0];
//~}
//~{
for (auto& dev : autd) {
  // do something
}
//~}
//~return 0; }

File: src/codes/Users_Manual/geometry_4.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = Controller.Open([new AUTD3()], new Nop());
~{
var dev = autd[0];
~}
foreach (var dev in autd)
{
  // do something
}